build-parent/pom.xml

```diff
@@ -20,6 +20,7 @@
     <version.jdk>11</version.jdk>
 
     <compiler-plugin.version>3.8.1</compiler-plugin.version>
+    <version.dependency.plugin>2.8</version.dependency.plugin>
```

Comment:
I couldn't use `mvn dependency:list`, `mvn dependency:tree` or `mvn dependency:analyze` because they were all failing with this error: `Plugin org.apache.maven.plugins:maven-dependency-plugin:${version.dependency.plugin} or one of its dependencies could not be resolved`, so I realized that the version wasn't specified anywhere in the repo.
Adding this line makes those commands work (at least in my local setup).

Reply:
This sounds strange to me, can you please create a ticket with the error you got and the fix so that we can double check?
It could be related to some wrong/mixed configuration at pom level 🤔

Reply:
Link to the [investigation task for this issue](https://issues.redhat.com/browse/MGDOBR-181).